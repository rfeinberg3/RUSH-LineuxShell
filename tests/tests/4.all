rush> ls: cannot access '/abc': No such file or directory
rush> Hello World
rush> 