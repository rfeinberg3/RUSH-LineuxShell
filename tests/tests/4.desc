Miscelaneous tests. Check if you program can handle multiple prompts correctly.
