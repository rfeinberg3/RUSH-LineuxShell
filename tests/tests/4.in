ls /abc
echo Hello World & sleep 1
exit
