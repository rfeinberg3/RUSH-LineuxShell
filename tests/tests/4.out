rush> rush> Hello World
rush> 